packages/@headlessui-react/src/components/combobox/combobox.tsx

```diff
@@ -364,7 +372,7 @@ type ComboboxValueProps<
   { nullable?: TNullable; multiple?: TMultiple }
 >
 
-type ComboboxProps<
+export type PropsCombobox<
```

Comment:
Any reason why you went with `PropsXXX` instead of `XXXProps` which seems to be more common in other libraries?

Personal note: I think it does read a little bit funny. Main advantage I see is in an import statement where you can use `import { Props<press space for code completion> } from '@headlessui/react'`

Reply:
Short answer: It was much simpler to do in a repetitive manner because it was consistent. I was also thinking about IDE support as you've suggested. I had also considered exposing additional prop types under `PropsOwnXXX` which includes only non-inherited / non-feature stuff as it's likely that's the _only_ thing people would be actually customizing in a wrapped-component situation. But I ended up not doing that.

This form means it's a simple regex operation to replace them all to swap them so I'll do that.